### PR TITLE
Interactive learning supercloud script

### PR DESCRIPTION
**Pull request by amburger66**
contains main approach and all baselines

### CODE REVIEW

File: scripts/supercloud/run_random_query_experiments.sh [hunk 1]
```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
```
Comment (tomsilver) [inline scripts/supercloud/run_random_query_experiments.sh:1]: so this is the script that you used to determine the 0.03 threshold, but we're not gonna use this script regularly going forward right? (this is fine and good to include still, just checking)

File: src/approaches/interactive_learning_approach.py [hunk 2]
```diff
@@ -391,7 +391,7 @@ def _query_policy(s: State) -> Optional[GroundAtomsHoldQuery]:
             atoms_to_query = set()
             for atom in ground_atoms:
                 score = self._score_atom_set({atom}, s)
-                if score > CFG.interactive_score_threshold:
+                if score > float(CFG.interactive_score_threshold):
```
Comment (tomsilver) [inline src/approaches/interactive_learning_approach.py:394]: why is this needed?

Comment (ronuchit) [inline src/approaches/interactive_learning_approach.py:394]: do we have a bug in our type checking for the CFG?

Comment (amburger66) [reply src/approaches/interactive_learning_approach.py:394]: before i didn't know how to handle floats in bash but yeah this shouldn't be necessary

Comment (amburger66) [reply src/approaches/interactive_learning_approach.py:394]: nah this should be unnecessary, will take it out

File: scripts/supercloud/run_random_query_experiments.sh (hunk 1 above)
Comment (amburger66) [reply scripts/supercloud/run_random_query_experiments.sh:1]: correct. also i verified that we can calculate the appropriate query probability based off of the environment and how many queries the main approach makes!

**Review by tomsilver — APPROVED** [review]
good stuff!
